_posts/2017-11-03-adapter-model.md

```diff
@@ -3,7 +3,7 @@ title: "Mixer Adapter Model"
 overview: Provides an overview of Istio 0.2's updated Mixer adapter model
 published: true
 permalink: blog/mixer-adapter-model.html
-attribution: The Istio Team
+attribution: The Mixer Crew
```

Comment:
Should this be called Istio-Mixer Crew ?, Just Mixer by itself can be confusing

Reply:
It's the Istio blog, so I think that's clear enough..

Reply:
Is "mixer crew" searchable on istio.io? Maybe a reference to the integrations WG would be more discoverable?